app/src/test/java/com/example/benchproject/test/common/LiveDataTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.benchproject.test.common
+
+import androidx.arch.core.executor.ArchTaskExecutor
+import androidx.arch.core.executor.TaskExecutor
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.jupiter.api.extension.AfterEachCallback
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+
+class LiveDataTest : BeforeEachCallback, AfterEachCallback {
+
+    override fun beforeEach(context: ExtensionContext?) {
+        Dispatchers.setMain(Dispatchers.Unconfined)
+        ArchTaskExecutor.getInstance()
+            .setDelegate(object : TaskExecutor() {
+                override fun executeOnDiskIO(runnable: Runnable) = runnable.run()
+
+                override fun postToMainThread(runnable: Runnable) = runnable.run()
+
+                override fun isMainThread(): Boolean = true
+            }
+            )
```

Comment:
Bad formatting, I don't know why ktlint has not failed

Reply:
Maybe now looks better? :D

Reply:
Probably such thing is not an error for ktlint, but it should look like this:
```
    override fun beforeEach(context: ExtensionContext?) {
        Dispatchers.setMain(Dispatchers.Unconfined)
        ArchTaskExecutor.getInstance()
            .setDelegate(
                object : TaskExecutor() {
                    override fun executeOnDiskIO(runnable: Runnable) = runnable.run()

                    override fun postToMainThread(runnable: Runnable) = runnable.run()

                    override fun isMainThread(): Boolean = true
                }
            )
    }
```

Reply:
Yes, I changed the code and looks the same